web/packages/extension/manifest.json

```diff
@@ -1,21 +1,32 @@
 {
     "manifest_version": 2,
     "name": "Ruffle",
-    "description": "Puts Flash back on the web where it belongs.",
+    "description": "__MSG_description__",
+    "default_locale": "en",
     "icons": {
         "16": "icons/16.png",
         "32": "icons/32.png",
         "48": "icons/48.png",
         "128": "icons/128.png",
         "180": "icons/180.png"
     },
+    "permissions": [ "storage" ],
     "content_scripts": [
         {
             "matches": ["<all_urls>"],
-            "js": ["js/lv0.js"],
+            "js": ["lv0.js"],
```

Comment:
Seems like this breaks the extension for me... The JS file is still in `dist/lv0.js`?

Reply:
My mistake.